fabric-events-lifecycle-v0/src/main/java/net/fabricmc/fabric/mixin/eventslifecycle/MixinInGameHud.java

```diff
@@ -0,0 +1,23 @@
+package net.fabricmc.fabric.mixin.eventslifecycle;
+
+import net.fabricmc.fabric.api.event.client.HudRenderCallback;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.gui.hud.InGameHud;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(InGameHud.class)
+public class MixinInGameHud {
+	@Shadow
+	@Final
+	private MinecraftClient client;
+
+	@Inject(method = "render", at = @At("RETURN"))
```

Comment:
Looking at this inection point, would it not be better to be done before:

```java
RenderSystem.color4f(1.0F, 1.0F, 1.0F, 1.0F);
RenderSystem.enableAlphaTest();
```

I dont think going at RETURN is necessarily the best location, however it is better than some fragile injectionpoint that could break silently in an update. Injecting after the `renderStatusEffectOverlay` call may even be a good place?